<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceFindWidget</class>
 <widget class="QWidget" name="SourceFindWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Find</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ui_find"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ui_replace"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="ui_matchCase">
       <property name="text">
        <string>Match Case</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ui_next">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/shortcuts/edit/icon_set/icons/arrow_redo.png</normaloff>:/shortcuts/edit/icon_set/icons/arrow_redo.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_replaceNext">
       <property name="text">
        <string>Replace and Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_replaceAll">
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_closeFind">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/shortcuts/file/icon_set/icons/cross.png</normaloff>:/shortcuts/file/icon_set/icons/cross.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/rc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ui_find</sender>
   <signal>returnPressed()</signal>
   <receiver>ui_next</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_closeFind</sender>
   <signal>clicked()</signal>
   <receiver>SourceFindWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
